docs/content/index.md

```diff
@@ -0,0 +1,47 @@
+# Getting Started with Piri
+
+This guide covers both the **Forge Production** and **Forge Staging** networks. The setup steps are identical except for two values. Use the **Environment** dropdown in the header to toggle the values shown below (defaults to Production).
+
+<div class="hero" markdown="1">
+  <div class="hero__logo">
+    <img src="images/logo.png" alt="Piri" width="40" height="40">
+  </div>
+  <div>
+    <p class="hero__title">Getting Started with Piri</p>
+    <p class="hero__subtitle">Pick your environment (Production/Staging) and deploy with the right flags.</p>
+  </div>
+</div>
+
+## Complete Setup Guide
+
+<ul class="step-card-grid">
+  <li class="step-card" markdown="1">
+    <p class="step-card__icon"><img src="images/icons/checklist.svg" alt="" width="18" height="18"></p>
+    <p class="step-card__title">Step 1: <a href="./setup/prerequisites">Prerequisites</a></p>
```

Comment:
These links do not work when viewing markdown in repo (need `.md` suffix) - can it be added such that it'll work in both renderings?